work-item-link-blackbox_test.go

```diff
@@ -519,6 +519,8 @@ func (s *workItemLinkSuite) TestShowWorkItemLinkOK() {
 	actual := link.WorkItemLink{}
 	require.Nil(s.T(), link.ConvertLinkToModel(*readIn, &actual))
 	require.True(s.T(), expected.Equal(actual))
+
+	require.NotNil(s.T(), readIn.Data.Links, "The link should include a self link")
```

Comment:
Asserting the presence of the "Self" attribute can be added, so you can add: `readIn.Data.Links.Self != ""`